docs/bodyswap.rst

```diff
@@ -13,16 +13,28 @@ Usage
 
 ::
 
-    bodyswap [--unit <id>]
+    bodyswap unit <id>
+    bodyswap linger
```

Comment:
This usage block should be:
```md
    bodyswap [--unit <id>]
    bodyswap linger
```

`--unit` is an option go `bodyswap` when it is run without a subcommand. `linger` is a subcommand.

Reply:
oh the -- part is no longer around cuz I dismantled the arg parse

Reply:
you still need argparse for the `--unit` option